testcases/open_posix_testsuite/conformance/interfaces/feof/1-1.c

```diff
@@ -0,0 +1,120 @@
+/*
+ * This file is licensed under the GPL license.  For the full content
+ * of this license, see the COPYING file at the top level of this
+ * source tree.
+ */
+
+/*
+ * assertion:
+ *
+ *	The feof function shall test the end-of-file indicator for
+ *	the stream pointed to by argument and the function shall return
+ *	non-zero if and only if the end-of-file indicator is set for stream
+ *
+ * method:
+ *	-open file in write mode
+ *	-write 2 sample strings into the file
+ *	-close the file
+ *	-open the same file in read mode
+ *	-read first string from file and check feof return value
+ *	-read second string from file and check feof return value
+ */
+
+#define _XOPEN_SOURCE 700
+#include <stdio.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include "posixtest.h"
+
+#define TNAME "feof/1-1.c"
+#define FNAME "in_file"
+#define STRING_CONST1 "LINUX"
+#define STRING_CONST2 "POSIX"
+
+static void remove_file(FILE *in_fp)
+{
+	errno = 0;
+	if (in_fp != NULL) {
+		if (fclose(in_fp) == EOF) {
+			printf(TNAME " Error at fclose(), errno = %d\n", errno);
+			unlink(FNAME);
+			exit(PTS_UNRESOLVED);
+		}
+	}
+	unlink(FNAME);
```

Comment:
There is absolutely no need to close the file before unlinking.

Reply:
Updated